### ISSUE TITLE
Hi! How many epochs should be trained according to your experience? (For X2)

### ISSUE THREAD

**Issue by Jian-danai** [xiezw5/Component-Divide-and-Conquer-for-Real-World-Image-Super-Resolution#16]
Hi! How many epochs should be used to train the model according to your experience?

I just want to reproduce the pre-trained model you provided. Thanks.

**Comment by Jian-danai** [comment]
Anybody find it hard to train 300 epochs(default setting in the code)? It seems to take about 15 days on a single GPU(Nvidia GTX TITAN).

**Comment by Jian-danai** [comment]
Final response from the authors: 

On both two dataset, we set the batch_size to 16 and the learning rate decays every 200000 iterations.
Both should be trained with 600000 iterations.

For RealSR, 200000 iterations=8000 epochs. For DRealSR, 200000 iterations=100 epochs.

Thanks for the response!!

**Comment by billhhh** [comment]
That's huge
